core/controller/cluster/controller_operation.go

```diff
@@ -272,6 +274,17 @@ func (c *controller) Rollback(ctx context.Context,
 		return nil, err
 	}
 
+	// Deprecated: for internal usage
+	if changed, err := c.manifestVersionChanged(ctx, application.Name, cluster.Name,
```

Comment:
wrap this as a function

Reply:
It could be more readable here. Only sync gitops branch when manifest version changed.